security/kernel-lockdown-tests.py

```diff
@@ -97,13 +97,11 @@ def test_lockdown_mem(self):
                 self.fail("'/dev/mem' file access permitted.")
 
     def test_lockdown_debugfs(self):
-        if self.distro_version.name == "SuSE":
-            self.cancel("This test not supported on SuSE")
         # Try read the values from sysfs file
         output = process.system_output('mount', ignore_status=True).decode()
         if 'debugfs' not in output:
             self.cancel("Skip this test as 'debugfs' not mounted.")
-        dbg_file = "/sys/kernel/debug/powerpc/xive"
+        dbg_file = "/sys/kernel/debug/powerpc/xive/store-eoi"
```

Comment:
is the path common for both RHEL and SUSE ?

Reply:
yes, common for both SLES and RHEL